lib/homePage.dart

```diff
@@ -90,17 +90,28 @@ class CategoryCard extends StatelessWidget {
           onTap: onPressed,
           child: Container(
               decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(30),
                   border: Border.all(width: 2), color: Colors.transparent),
               width: MediaQuery.of(context).size.width,
               height: MediaQuery.of(context).size.width * 0.5,
               child: Column(
                 crossAxisAlignment: CrossAxisAlignment.center,
                 mainAxisAlignment: MainAxisAlignment.center,
                 children: [
-                  SvgPicture.asset(
-                    icon,
-                    height: 40.0,
-                    width: 40.0,
+                  ShaderMask(
```

Comment:
And this color also looks amazing🔥, but the theme of the application is Black and white so I think black and white would look nice! what do you think?